cockroachdb/values.yaml

```diff
@@ -134,6 +134,16 @@ conf:
   # CockroachDB's port to listen to HTTP requests.
   http-port: 8080
 
+  # CockroachDB's storage configuration https://www.cockroachlabs.com/docs/v20.2/cockroach-start.html#storage
+  store:
+    enabled: false
+    # Should be empty or 'mem'
+    type:
+    path: cockroach-data
+    # Required for type=mem. If type and size is empty - storage.persistentVolume.size is used
+    size:
+    # hdd:7200rpm
+    attrs:
```

Comment:
Can you add a short description for each of these attributes to the [configuration section of the README.md](https://github.com/cockroachdb/helm-charts/blob/master/cockroachdb/README.md#configuration)?